src/Controller/HomeController.php

```diff
@@ -32,4 +35,23 @@ public function faq(): Response
     {
         return $this->render('home/faq.html.twig');
     }
+
+    #[Route('/contact', name: 'app_contact')]
+    public function contact(): Response
+    {
+        return $this->render('home/contact.html.twig');
+    }
+
+    #[Route('/contact/email', name: 'app_contact_email')]
+    public function email(MailerInterface $mailer, Request $request): Response
+    {
+        $email = (new Email())
+            ->from($request->request->get('email'))
+            ->to('hureaux.lucas@gmail.com')
```

Comment:
Ça me semble inversé : le mail est à envoyer "to" le mail indiqué dans le formulaire, "from" le mail de référence de l'entité qui administre le site (qui n'est pas ton mail)